src/utils/helpers/application/getSectionsDetails.ts

```diff
@@ -26,7 +26,7 @@ const getSectionDetailsArray = (sectionsList: TemplateSection[]): SectionDetails
     const totalPages = pageBreaks.length + 1
 
     return {
-      active: true,
+      active: false,
```

Comment:
Sorry, I wasn't actually suggesting you change this, just asking a question. Not sure what the implications are, just go with whichever looks best as it loads, I guess.

Reply:
I think this is probably best indeed. It's just the default, before it checks the elements in the page to set an `active` flag for the sections, so in case there isn't any element - which shouldn't happen, it would be active = false. And that seems more accurate to me.